src/app/features/public-form/public-form-config.ts

```diff
@@ -1,12 +1,13 @@
 import { Entity } from "../../core/entity/model/entity";
 import { DatabaseEntity } from "../../core/entity/database-entity.decorator";
 import { DatabaseField } from "../../core/entity/database-field.decorator";
+import { FieldGroup } from "app/core/entity-details/form/field-group";
 
 @DatabaseEntity("PublicFormConfig")
 export class PublicFormConfig extends Entity {
   @DatabaseField() title: string;
   @DatabaseField() description: string;
   @DatabaseField() entity: string;
-  @DatabaseField() columns: string[][];
+  @DatabaseField() columns: FieldGroup;
```

Comment:
I actually had thought to get `columns: FieldGroup[]` here (aligning exactly with `FormConfig` interface for the details views). Do you want to still quickly adapt this, @Abhinegi2 ?

Reply:
Sure Sebastian will work on this.

Reply:
made the requested changes please have a look now-

- [ ] update the type columns: FieldGroup[]
- [ ] Added migration for the default values that are defined as "prefilled"